mep/common/models.py

```diff
@@ -90,16 +90,32 @@ class Meta:
     def dates(self):
         '''Date or date range as a string for display'''
 
-        # if no dates are set, return an empty string
-        if not self.start_year and not self.end_year:
+        if not self.start_year and not self.end_year: # no dates are set
             return ''
 
-        # if start and end year are the same just return one year
-        if self.start_year == self.end_year:
-            return self.start_year
+        if not self.end_year: # only start year
+            if self.start_year < 0: # and it's BCE
+                return str(abs(self.start_year)) + ' BCE-' # '100 BCE-'
```

Comment:
I'd revise all of these to use string formatting rather than concatenation. 
```return  '%s BCE-' % abs(self.start_year)```

That way too you let `%s` interpolate it as a string rather than having to call `str`.

Reply:
ah, that's smart, thank you!